backend/src/main/resources/db/migration/V2.3__make_user_id_nullable.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `member`
+    MODIFY COLUMN `user_id` VARCHAR(255);
```

Comment:
`workspace_id`도 nullable해야하지 않나요?

```suggestion
ALTER TABLE `member`
    MODIFY COLUMN `user_id` VARCHAR(255);
ALTER TABLE `member`
    MODIFY COLUMN `workspace_id` VARCHAR(255);    
```

Reply:
클래스에서는 (default)nullable=true로 되어있어서 인지를 못했네요, 반영하겠습니다!